xml/System.Diagnostics/ActivityContext.xml

```diff
@@ -18,7 +18,7 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>A representation that conforms to the W3C TraceContext specification. It contains two identifiers: a TraceId and a SpanId,  along with a set of common TraceFlags and system-specific TraceState values.</summary>
     <remarks>To be added.</remarks>
```

Comment:
><remarks>To be added.</remarks> [](start = 4, length = 31)

Please add the remark saying:

ActivityContext contains the property IsRemote which is not part of W3C. IsRemote is indicating if the context is propagated from a remote parent

Reply:
```suggestion
    <remarks>
      <format type="text/markdown"><![CDATA[

## Remarks

ActivityContext contains the property `IsRemote`, which is not part of W3C. `IsRemote` is indicating if the context is propagated from a remote parent.

      ]]></format>
    </remarks>
```